src/rust/ensogl/lib/components/src/card.rs

```diff
@@ -0,0 +1,71 @@
+//! This module provides the [`Card`] component.
```

Comment:
But it explains, doesn't it? This is the module with `Card` component. This is the point of the module.

Personally:
1. I would avoid duplicate descriptions of `Card` in `Card`'s docs and module's docs; otherwise there's high chance of them being desynchronized.
2. I prefer having main explanation of `Ca in structure, not in module. This is how I usually check documentation in rust: rather go directly to structure I use, or want to use, or see used. Rarely I jump to the module.

Reply:
@farmaazon there sohuld be at least 1 sentence description of what a card is. Card is a background with rounded corners shared between different components. Detailed doc can be on the struct, but the file docs should explain it already in short way.

Reply:
The description is included now.